recipes/pyocd/meta.yaml

```
@@ -0,0 +1,62 @@
+{% set name = "pyocd" %}
+{% set version = "0.34.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pyocd-{{ version }}.tar.gz
+  sha256: dbc6eb231dc6ada9ff53f493d26fa7a8f6d315d4e641783eceb1dc4a79e240ac
+
+build:
+  skip: true  # [py2k]
+  entry_points:
+    - pyocd = pyocd.__main__:main
+    - pyocd-gdbserver = pyocd.tools.gdb_server:main
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - capstone >=4.0,<5.0
+    - cmsis-pack-manager >=0.4.0,<1.0
+    - colorama <1.0
+    - dataclasses  # [py<37]
+    - hidapi >=0.10.1,<1.0
+    - intelhex >=2.0,<3.0 
+    - intervaltree >=3.0.2,<4.0
+    - libusb-package >=1.0,<2.0
+    - natsort >=8.0.0,<9.0
+    - prettytable >=2.0,<4.0
+    - pyelftools <1.0
+    - pylink-square >=0.11.1,<1.0
+    - python
+    - pyusb >=1.2.1,<2.0
+    - pyyaml >=6.0,<7.0
+    - six >=1.15.0,<2.0
+    - typing-extensions >=4.0,<5.0
```

```suggestion
requirements:
  host:
    - pip
    - python >=3.6
  run:
    - capstone >=4.0,<5.0
    - cmsis-pack-manager >=0.4.0,<1.0
    - colorama <1.0
    - dataclasses
    - hidapi >=0.10.1,<1.0
    - intelhex >=2.0,<3.0 
    - intervaltree >=3.0.2,<4.0
    - libusb-package >=1.0,<2.0
    - natsort >=8.0.0,<9.0
    - prettytable >=2.0,<4.0
    - pyelftools <1.0
    - pylink-square >=0.11.1,<1.0
    - python >=3.6
    - pyusb >=1.2.1,<2.0
    - pyyaml >=6.0,<7.0
    - six >=1.15.0,<2.0
    - typing-extensions >=4.0,<5.0
```
https://github.com/pyocd/pyOCD/blob/800abc3dd6f05de890755a542677dabc13296574/setup.cfg#L47

Also, the dataclasses package can be installed on later python versions.